example.js

```diff
@@ -4,7 +4,87 @@ const fetch = require('.');
 
 app.on('window-all-closed', () => {});
 
+const p = [];
+
+// Basic example
+p.push((async () => {
+	console.log('basic example');
+
+	const response = await fetch('https://api.ipify.org');
+	console.log('my ip', await response.text());
+
+	for (const [key, value] of response.headers.entries()) {
+		console.log('header', key, value);
+	}
+
+	console.log('response', response);
+}));
+
+// Body used
+p.push((async () => {
+	console.log('body used example');
+
+	const response = await fetch('https://api.ipify.org');
+
+	console.log('body used?', response.bodyUsed);
+	console.log('my ip', await response.text());
+	console.log('body used?', response.bodyUsed);
+}));
+
+// Body lock
+p.push((async () => {
+	console.log('body lock example');
+
+	const response = await fetch('https://api.ipify.org');
+	console.log('my ip', await response.text());
+
+	try {
+		console.log('my ip', await response.text());
+	} catch (error) {
+		// Expected error because body was already used
+		console.error(error);
+	}
+}));
+
 (async () => {
-	console.log('Your IP:', await fetch('https://api.ipify.org', {type: 'text'}));
+	for (let i = 0; i < p.length; i++) {
+		console.log('=========');
+		await p[i]();
+	}
+
+	app.quit();
+})();
+
+/* Test example
```

Comment:
Why is this commented out?

Reply:
I was having troubles doing tests with electron + ava, it's solved now.